CHANGES.md

```diff
@@ -71,6 +71,8 @@ Format of the entries must be.
 
 * Mesos now uses the jemalloc memory profiler by default. (DCOS_OSS-2137)
 
+* Updated DC/OS UI to master+v2.3.0 [Changelog](https://github.com/dcos/dcos-ui/releases)
```

Comment:
shall we point the url to a specific release?

Reply:
I don' think that's possible, there doesn't seem to be an anchor tag

Reply:
https://github.com/dcos/dcos-ui/releases/tag/master%2Bv2.3.0

Reply:
Cool. I missed that 👍

Reply:
lets discuss this on EE PR to not have the same discussion twice

Reply:
Let's please at least add the old tag `Updated DC/OS UI from v1.12.0-rc.1 to master+v2.3.0` so it is easier navigating the change log